c/httpfileservice.c

```diff
@@ -60,10 +60,13 @@ void response200WithMessage(HttpResponse *response, char *msg) {
   finishResponse(response);
 }
 
-/* Returns a boolean value for whether or not
- * the specified file is a directory or not.
+/* Returns an int value for whether or not the specified
+ * path is a file, folder, or it doesn't exist.
+ * -1 Does not exist
+ * 0 File exists 
+ * 1 Folder exists
  */
-bool isDir(char *absolutePath) {
+int doesFileOrDirectoryExist(char *absolutePath) {
```

Comment:
* Why did you change an existing global function?
* Why does a file related global function have to be in ```httpfileservice.c```? It has nothing to do with http.

Reply:
- I changed isDir because it didn't work. It was a bool that returned 3 things. I also changed the name because the name didn't make much sense compared to what it was trying to do. So either keep 2 methods that are bools, or make 1 int method return 3 conditions, I chose the second option. It was also being used in one place, and thus will not break things because I also updated where it was being used.

- I don't have answer for that, that's just where it was before, didn't want to change stuff up. I guess I could move it to unixFileService.c instead if that's useful?

Reply:
* ```bool isDir(const char*)``` should tell whether the specified file is a directory. If it doesn't, then there is a bug. From this name, I only expect that. If the path doesn't exist, it's not a directory. The name should drive the implementation in this case, that's an existing function.
* You can make a new static function to do what you need to do. If anyone needs that outside this file, they'll make it global.

Reply:
Right @ifakhrutdinov and it didn't work, which is why it was a bug. So I made it work and renamed it to a name that is more appropriate for what it does. If you want instead, I could keep "isDir" and then remove the -1 part, thereby removing the third condition and make it a true "bool" if that's a good thing to do, but I don't see the point of it when 2 methods can just be 1 for simplicity's sake.

I can keep the original method and make another, whatever works best I'll incorporate